packages/flutter_tools/lib/src/project.dart

```diff
@@ -636,6 +636,9 @@ class WebProject {
   /// The 'lib' directory for the application.
   Directory get libDirectory => parent.directory.childDirectory('lib');
 
+  /// The directory containing additional html files for the application,
```

Comment:
nit: could be any static files, no? Also, you probably meant to end with a period, not a comma.

Reply:
Fixed